edge-hub/watchdog/src/main.rs

```diff
@@ -40,19 +44,25 @@ fn main() -> Result<()> {
         Arc::clone(&should_shutdown),
     )?;
 
-    let broker_handle = match run(
-        "MQTT Broker",
-        "/usr/local/bin/mqttd",
-        vec!["-c".to_string(), "/app/mqttd/broker.json".to_string()],
-        Arc::clone(&should_shutdown),
-    ) {
-        Ok(handle) => Some(handle),
-        Err(e) => {
-            should_shutdown.store(true, Ordering::Relaxed);
-            error!("Could not start MQTT Broker process. {}", e);
-            None
-        }
-    };
+    let mut broker_handle = None;
```

Comment:
Something like this might be better. Check with @dmolokanov  
let broker_handle = match env::var("experimentalFeatures:mqttBrokerEnabled").ok().filter(|v| v =="true")
    {
        Some(_) => ...,
        None => None,
    };